.github/workflows/release-periodically.yml

```diff
@@ -20,13 +20,11 @@ on:
     # Tuesdays are the traditional rawhide anaconda release day, so that there are enough days to
     # unbreak things.
 
-permissions:
-  contents: write
-
```

Comment:
I wonder why you are removing permissions? By doing that you will increase permissions for this workflow and also you need some to Checkout the repository.

Reply:
Oh. I thought the token permission would override this. But you are right, the token is used only for the one step.

Reply:
Removed the removal.

"Odvolávám co jsem odvolal..."